src/lib/models/role-v2.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
I wonder if we should rename this ContactRole and the other one DocumentRole if we are going to have different role models for the various jobs?
Alternatively the original Role may be flexible enough if it was sparsely populated?